nservicebus/pipeline/customizing-error-handling_content_core_[8.0,).partial.md

```diff
@@ -0,0 +1,33 @@
+Custom exception handling can be implemented using pipeline behaviors. To learn more about pipeline and behaviors refer to the documentation on [how to manipulate the pipeline with behaviors](/nservicebus/pipeline/manipulate-with-behaviors.md).
+
+NOTE: Behaviors are meant to customize recoverability i.e. how many times and under which conditions a message should be re-tried. To read more about customizing recoverability refer to the [recoverability documentation](/nservicebus/recoverability/).
```

Comment:
> NOTE: Behaviors are meant to customize recoverability i.e. how many times and under which conditions a message should be re-tried.

why would we say something like that? shouldn't this be all part of the recoverability configuration?